src/bag/models.py

```diff
@@ -82,12 +82,12 @@ class Meta:
 class Openbareruimte(GeoModel):
     id = models.CharField(max_length=16, primary_key=True, db_column="identificatie")
     document_mutatie = models.DateField(db_column="documentdatum")
-    document_nummer = models.CharField(max_length=20, db_column="documentnummer")
+    document_nummer = models.CharField(max_length=40, db_column="documentnummer") # FIXME: Remove after problem in sources for id: 0363300011951944 is fixed.
```

Comment:
Ik weet niet helemaal wat het probleem is, maar kunnen we die record er niet gewoon uit weren? Dan hoeven we geen DB migration te doen voor een rogue record.

Reply:
Inmiddels gefixed, dus ik zal hem er weer uithalen. Eruit halen is moeilijk omdat je dan ook de foreign key relations uit de andere tabellen halen